pkgs/games/anki/default.nix

```diff
@@ -22,14 +26,18 @@
 , plotsSupport ? true
 }:
 
-let
-    # Development version of anki has bumped to beautifulsoup4
-    beautifulsoup = callPackage ./beautifulsoup.nix { };
-
-    qt4 = pyqt4.qt;
-
+let qt5 = qt59;
+    pyqt = pyqt5.override {
+      qmake = qt5.qmake;
+      qtbase = qt5.qtbase;
+      qtsvg = qt5.qtsvg;
+      qtwebkit = qt5.qtwebkit;
+      qtwebengine = qt5.qtwebengine;
+      qtwebsockets = qt5.qtwebsockets;
+      qtconnectivity = qt5.qtconnectivity;
+    };
```

Comment:
Why are these overrides necessary?

Reply:
ah different qt version.

Reply:
So 5.10 does not work here? I am suprised that one can link an older version of qt against pyqt.

Reply:
Traceback (most recent call last):
  File "/nix/store/f4ad9cx0fil9qq6zvjk4x5bqsrzgvhmi-anki-2.1.5/bin/..anki-wrapped-wrapped", line 4, in <module>
    import aqt
  File "/nix/store/f4ad9cx0fil9qq6zvjk4x5bqsrzgvhmi-anki-2.1.5/lib/python3.6/site-packages/aqt/__init__.py", line 14, in <module>
    from aqt.qt import *
  File "/nix/store/f4ad9cx0fil9qq6zvjk4x5bqsrzgvhmi-anki-2.1.5/lib/python3.6/site-packages/aqt/qt.py", line 42, in <module>
    raise Exception("Anki only supports Qt 5.9.x at this time.")
Exception: Anki only supports Qt 5.9.x at this time.

Reply:
Can we temporary upgrade to anki 2.1.6-beta1 instead of compiling pyqt5 just for anki?